scania-digital-design-system/tegel
# fix(table): sorting arrow default and direction correction

## **Describe pull-request**  
Fixing two issue:
 - Missing default/unsorted icon back to unsorted columns
 - Changing arrow direction for asc and desc directions

Improvement:
 - Replacing hard-coded icons with icon components

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3256](https://tegel.atlassian.net/browse/CDEP-3256)    
- **GitHub:** https://github.com/scania-digital-design-system/tegel/issues/596

## **How to test**  
1. Open preview link, go to sortable table story
2. On initial state, all unsorted columns on hover should have icon with two arrows, one pointing down, one up to indicate this columns is sortable.
3. Clicking on header, ascending has arrow pointing up, descending has arrow pointing down.

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
_Include before/after screenshots for UI changes._

## **Additional context**  



[CDEP-3256]: https://tegel.atlassian.net/browse/CDEP-3256?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
